concent_api/core/tests/test_unit_model.py

```diff
@@ -17,7 +16,6 @@
 from core.constants import ETHEREUM_TRANSACTION_HASH_LENGTH
 from core.constants import MOCK_TRANSACTION
 from core.message_handlers import store_subtask
-from core.models import Client
```

Comment:
And in another commit you delete this. Please fix it

Reply:
it is fixed in next commit. Yes, I agree it should't be done like this. But is it really necessary to waste developers time to repair history, when only one benefit with this is that it will be "nicer" in git-cola? Code should be clean, easy to reed, quality should be good- but fixes like this??

Reply:
@pawelkisielewicz as I generally agree that we waste too much time "polishing" the history, I must highlight that we have agreed on specific way of working and until we change our agreement we need to stick to it.

Reply:
ok